packages/client/src/components/app/forms/StringField.svelte

```diff
@@ -12,6 +12,7 @@
   export let defaultValue = ""
   export let align
   export let onChange
+  export let runOnInput = false
```

Comment:
Why false, should it not be undefined?

Reply:
It's a boolean setting (you can check the manifest to confirm)  with a default value of false, so should be initialised to `false` here too.